android/jni/app/organicmaps/platform/Language.cpp

```diff
@@ -50,8 +50,18 @@ Locale GetCurrentLocale()
                                                                     "()Ljava/lang/String;");
   jni::ScopedLocalRef currencyCode(env, env->CallStaticObjectMethod(g_utilsClazz, getCurrencyCodeId));
 
+  static jmethodID const getDecimalSeparatorId = jni::GetStaticMethodID(env, g_utilsClazz, "getDecimalSeparator",
+                                                                    "()Ljava/lang/String;");
```

Comment:
Align it with (

Reply:
Noted! Commit will be amended accordingly...